packages/websocket/package.json

```
@@ -1,10 +1,30 @@
 {
   "name": "@solid-primitives/websocket",
-  "version": "0.3.7",
-  "description": "Primitive to create a worker helper",
+  "version": "0.4.0",
```

```suggestion
  "version": "0.3.7",
```
let the changeset do the work ;)